pkg/yurthub/filter/responsefilter/filter.go

```diff
@@ -179,11 +179,15 @@ func (frc *filterReadCloser) streamResponseFilter(rc io.ReadCloser, ch chan *byt
 			return err
 		}
 
-		newObj := obj
+		newObj := obj.DeepCopyObject()
```

Comment:
Is it necessary to `DeepCopyObject`?

Reply:
yes, there is really no need to DeepCopyObject. i will fix it.